test/prism_regression/assign_to_constant_path.rb

```diff
@@ -0,0 +1,5 @@
+# typed: false
+
+# Regular assignment
+ConstantPath::RegularAssignment = 1
+::FullyQualified::ConstantPath::RegularAssignment = 2
```

Comment:
Really minor, but I think it'd be more understandable if these to live in [`assign_to_const.rb`](https://github.com/Shopify/sorbet/pull/236/files#diff-3f366b30dda812f3e201a4864a1dc5462d3cefb6c6fec1132c28cac4f3c59467). The fact that they're different node types is non-obvious, and seeing the contrast between the two on the same file could be useful to future readers